README.md

```diff
@@ -20,9 +20,24 @@
 
 ## Usage
 
+### Full Import
+
 ```bash
+# Choose a package manager you like.
+
+# NPM
 npm i onu-ui
 npm i unocss -D
+
+# Yarn
+yarn add onu-ui
+yarn add unocss -D
+
+
+# pnpm
+pnpm add onu-ui
+npm add unocss -D
+
```

Comment:
I don't think we need so much information, they know what package manager to use.

Reply:
yeah, you can delete this part.

Reply:
done